### DISCUSSION TITLE
Allow specifying a CA certificate for Redis

### DISCUSSION THREAD

**Discussion by norman-zon** [renovatebot/renovate#21450] [Request Help] [answered]
### Type of discussion.

I'm proposing an idea

### Tell us more.

I would like to use an external Redis instance by setting `RENOVATE_REDIS_URL`, that uses secure connections with TLS and a self-signed certificate.
Therefore it would be necessary to be able to specify a location of a `ca.cert` that gets used when establishing the Redis connection using a `rediss://` URL. I would prefer to use a env variable like `RENOVATE_REDIS_CA` for this.

**Comment by rarkins** [comment]
How do you run Renovate? And is Redis the only server you connect to which requires the self signed certificate? e.g. you don't use it on github or gitlab, on artifactory, etc?

**Comment by norman-zon** [comment]
I run it self-hosted via the helm chart.
Redis is the only connection that requires the cert. We use Google Cloud Redis, which [brings its own certs](https://cloud.google.com/memorystore/docs/redis/about-in-transit-encryption#certificate_authority).

**Reply by rarkins** [reply]
Have you looked into `NODE_EXTRA_CA_CERTS`?

**Reply by norman-zon** [reply, marked as answer]
Oh, I misinterpreted the docs on `NODE_EXTRA_CA_CERTS` on my first read-through. I thought I had to build the certificate into my image. 
Looks like `NODE_EXTRA_CA_CERTS` does exactly what I want. Thanks!
